### DISCUSSION TITLE
how to run source code in my VSCode？

### DISCUSSION THREAD

**Discussion by chenshuichuan** [Budibase/budibase#14358] [Help] [answered]
I have read some official documents, but I haven't found a way to run the source code？
is anyone know about it？thanks a lot！

**Comment by mike12345567** [comment, marked as answer]
Hi @chenshuichuan - you can follow the docs here to run the monorepo services on your machine: https://github.com/Budibase/budibase/blob/master/docs/CONTRIBUTING.md#contributing-to-budibase

**Reply by chenshuichuan** [reply]
got it， thank you very much！
